src/frontend-samples/fire-decoration-sample/README.md

```diff
@@ -2,14 +2,14 @@
 
 Copyright © Bentley Systems, Incorporated. All rights reserved.
 
-This samples show how to create a particle effect using fire as an example.
+This sample show how to create a particle effect using fire as an example.
```

Comment:
/s/show/shows. There are extensions for VSCode that can catch these kinds of mistakes.